tests/dataframe/test_groupby_pytest.py

```diff
@@ -230,3 +230,19 @@ def test_groupby_mode(self):
         match = "Currently mode is not supported for groupby"
         with pytest.raises(NotImplementedError, match=match):
             ed_flights.groupby("Cancelled").mode()
+
+    @pytest.mark.parametrize("dropna", [True, False])
+    @pytest.mark.parametrize("columns", ["Cancelled", ["dayOfWeek", "Cancelled"]])
+    @pytest.mark.parametrize("quantiles", [0.55, [0.2, 0.4, 0.6, 0.8]])
+    def test_groupby_aggs_quantile(self, dropna, columns, quantiles):
```

Comment:
Can we also add a test for an empty call like this: `.quantile()`?

Reply:
Yes :)